misc/distributed-bulkload-example/AWS/AWS-Bulkload.md

```diff
@@ -1,2 +1,82 @@
 # Distributed AWS Bulkload Example
-WIP
\ No newline at end of file
+
+1. Configure Aerolab for AWS
```

Comment:
When using numbers in .md its best to put the entire content of the number with a tab